src/pages/accountsettings/AccountSettings.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, Component } from "react";
+import { Link } from "react-router-dom";
+// import "../signup/signup.css";
+
+import "../../common.css"
+
+const initialValues = {
+  email: "",
+  password: "",
+};
+
+function AccountSettings() {
+  const [component, setComponent] = useState("signup");
+
+  // States for registration
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [phoneNumber, setPhoneNumber] = useState("");
+  const [userName, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [repeatPassword, setRepeatPassword] = useState("");
+
+  // Handling the name change
+  const handleName = (e) => {
+    setName(e.target.value);
+  };
+
+  // Handling the phone number change
+  const handlePhoneNumber = (e) => {
+    setPhoneNumber(e.target.value);
+  };
+
+  // Handling the email change
+  const handleEmail = (e) => {
+    setEmail(e.target.value);
+  };
+
+  // Handling the username change
+  const handleUserName = (e) => {
+    setUsername(e.target.value);
+  };
+
+  // Handling the password change
+  const handlePassword = (e) => {
+    setPassword(e.target.value);
+  };
+
+  // Handling the repeat password change
+  const handleRepeatPassword = (e) => {
+    setRepeatPassword(e.target.value);
+  };
+
+  return (
+    <div className="content-container">
+      <div className="hero-text">
+          <h1>StudentIT</h1>
+          <h3>Edit your account</h3>
+      </div>
+
+      <div className="input-wrapper">
+        <h1 className="input-header">Account settings</h1>
+
+        <div className="box-input-container sign-up-container">
+
+          <div className="input-container">
+            <h3>Full Name</h3>
+            <input placeholder="Type here..." onChange={handleName} value={name}></input>
+          </div>
+
+          <div className="input-container">
+            <h3>Email</h3>
+            <input placeholder="Type here..." onChange={handleEmail} value={email}></input>
+          </div>
+
+          <div className="input-container">
+            <h3>User Name</h3>
+            <input placeholder="Type here..." onChange={handleUserName} value={userName}></input>
+          </div>
+
+          <div className="input-container">
+            <h3>Phone Number</h3>
+            <input placeholder="Type here..." onChange={handlePhoneNumber} value={phoneNumber}></input>
+          </div>
+
+          <div className="input-container">
+            <h3>Password</h3>
+            <input placeholder="Type here..." onChange={handlePassword} value={password}></input>
```

Comment:
The input field from HTML comes with an attribute called `type`. And this type can accept a value called `password`. This will give your input behavior for a password, and it will replace your letters with stars. 

Same with all inputs above, give them a type. For the email as well.

![Screenshot 2022-11-14 at 20 42 39](https://user-images.githubusercontent.com/58588711/201757325-3cee7358-2633-40ef-ba3f-e0fce4b05cc8.png)